quic/s2n-quic/Cargo.toml

```
@@ -52,12 +52,12 @@ rand = "0.8"
 rand_chacha = "0.3"
 ring = { version = "0.16", optional = true, default-features = false }
 s2n-codec = { version = "=0.1.0", path = "../../common/s2n-codec" }
-s2n-quic-core = { version = "=0.7.1", path = "../s2n-quic-core" }
-s2n-quic-platform = { version = "=0.7.1", path = "../s2n-quic-platform", features = ["tokio-runtime"] }
-s2n-quic-rustls = { version = "=0.7.1", path = "../s2n-quic-rustls", optional = true }
-s2n-quic-tls = { version = "=0.7.1", path = "../s2n-quic-tls", optional = true }
-s2n-quic-tls-default = { version = "=0.7.1", path = "../s2n-quic-tls-default", optional = true }
-s2n-quic-transport = { version = "=0.7.1", path = "../s2n-quic-transport" }
+s2n-quic-core = { version = "=0.8.0", path = "../s2n-quic-core" }
+s2n-quic-platform = { version = "=0.7.2", path = "../s2n-quic-platform", features = ["tokio-runtime"] }
+s2n-quic-rustls = { version = "=0.7.2", path = "../s2n-quic-rustls", optional = true }
+s2n-quic-tls = { version = "=0.7.2", path = "../s2n-quic-tls", optional = true }
+s2n-quic-tls-default = { version = "=0.7.2", path = "../s2n-quic-tls-default", optional = true }
+s2n-quic-transport = { version = "=0.7.2", path = "../s2n-quic-transport" }
```

```suggestion
s2n-quic-core = { version = "=0.8.0", path = "../s2n-quic-core" }
s2n-quic-platform = { version = "=0.8.0", path = "../s2n-quic-platform", features = ["tokio-runtime"] }
s2n-quic-rustls = { version = "=0.8.0", path = "../s2n-quic-rustls", optional = true }
s2n-quic-tls = { version = "=0.8.0", path = "../s2n-quic-tls", optional = true }
s2n-quic-tls-default = { version = "=0.8.0", path = "../s2n-quic-tls-default", optional = true }
s2n-quic-transport = { version = "=0.8.0", path = "../s2n-quic-transport" }
```
this should look like above